### PR TITLE
ADD: `can.re`

### PR DESCRIPTION
**Pull request by OsSav**
### Checklist of required steps

* [x] Description of Organization
* [x] Robust Reason for PSL Inclusion
* [x] DNS verification via dig
* [x] Run Syntax Checker (make test)

* [x] Each domain listed in the PRIVATE section has and shall maintain at least two years remaining on registration, and we shall keep the \_PSL txt record in place in the respective zone(s) in the affected section

__Submitter affirms the following:__ 

  * [x] We are listing *any* third-party limits that we seek to work around in our rationale such as those between IOS 14.5+ and Facebook (see [Issue #1245](https://github.com/publicsuffix/list/issues/1245) as a well-documented example)

  * [x] This request was _not_ submitted with the objective of working around other third-party limits

  * [x] The [Guidelines](https://github.com/publicsuffix/list/wiki/Guidelines) were carefully _read_ and _understood_, and this request conforms
  * [x] The submission follows the [guidelines](https://github.com/publicsuffix/list/wiki/Format) on formatting and sorting

---

For Private section requests that are submitting entries for domains that match their organization website's primary domain, please understand that this can have impacts that may not match the desired outcome and take a long time to rollback, if at all.

To ensure that requested changes are entirely intentional, make sure that you read the affectation and propagation expectations, that you understand them, and confirm this understanding. 

PR Rollbacks have lower priority, and the volunteers are unable to control when or if browsers or other parties using the PSL will refresh or update.

(Link: [about propagation/expectations](https://github.com/publicsuffix/list/wiki/Guidelines#appropriate-expectations-on-derivative-propagation-use-or-inclusion))

 * [x] *Yes, I understand*.  I could break my organization's website cookies etc. and the rollback timing, etc is acceptable.  *Proceed*.
---

Description of Organization
====

can.re - Expiry Date: 2031 year,
official e-mail [support@ossav.com](mailto:support@ossav.com) always works.
Website: https://nic.can.re/ (under construction)
We offer various Internet services for our customers. We create new zone " .can.re", so we would like these zone to be on the public suffix list as well as others zones " .(*).re ".
Organization Name: OsSav Technology Ltd.
Organization Website: https://ossav.com
Number of users this request is being made to serve:
Current: 1560+
Estimated in one year:: 15000+

Reason for PSL Inclusion
====

Users can register domains in the domain can.re.
which means subdomains (*.can.re) are registered by independent 3rd parties.
We want user subdomains to be isolated from each other (cookies, suffix allocation, etc.). The domain is registered until 2031 year.

Raports:
https://dnssor.com/nic.can.re/whois
https://toolbox.googleapps.com/apps/dig/#NS/nic.can.re
https://www.google.com/search?q=site%3A*.can.re
https://dnssor.com/nic.can.re
https://intodns.com/nic.can.re
https://toolbox.googleapps.com/apps/dig/#NS/nic.can.re
https://transparencyreport.google.com/safe-browsing/search?url=can.re
https://en.wikipedia.org/wiki/.re
Whois: https://dnssor.com/nic.can.re/whois
DNS Report: https://intodns.com/nic.can.re
Full DNS Report: https://dnssor.com/nic.can.re
History: https://dnssor.com/nic.can.re/history

DNS Verification via dig
=======
```
dig +short TXT _psl.can.re
"https://github.com/publicsuffix/list/pull/1651"
```

Results of Syntax Checker (`make test`)
=========
```
============================================================================
Testsuite summary for libpsl 0.21.1
============================================================================
# TOTAL: 3
# PASS:  3
# SKIP:  0
# XFAIL: 0
# FAIL:  0
# XPASS: 0
# ERROR: 0
============================================================================
Making check in tests
  CC       test-is-public.o
  CC       test-is-public-all.o
  CC       test-is-cookie-domain-acceptable.o
  CC       test-is-public-builtin.o
  CC       test-registrable-domain.o
  CCLD     test-is-cookie-domain-acceptable
  CCLD     test-is-public-builtin
  CCLD     test-is-public
  CCLD     test-is-public-all
  CCLD     test-registrable-domain
PASS: test-is-public-builtin
PASS: test-is-public
PASS: test-is-cookie-domain-acceptable
PASS: test-registrable-domain
PASS: test-is-public-all
============================================================================
Testsuite summary for libpsl 0.21.1
============================================================================
# TOTAL: 5
# PASS:  5
# SKIP:  0
# XFAIL: 0
# FAIL:  0
# XPASS: 0
# ERROR: 0
============================================================================
Making check in msvc
```

### CODE REVIEW

**Review by dnsguru — CHANGES_REQUESTED** [review]
Thank you for resubmitting.  Can you place this higher in the #PRIVATE section of the file, so that it is sorted alphabetically by your org name among the other org names instead of at end of the file?

**Comment by OsSav** [comment]
> Thank you for resubmitting. Can you place this higher in the #PRIVATE section of the file, so that it is sorted alphabetically by your org name among the other org names instead of at end of the file?

694.line added:
// can.re : https://nic.can.re/
// Whois Server : whois.can.re
// Submitted by OsSav Technology Ltd. : support@ossav.com
can.re

**Review by dnsguru — CHANGES_REQUESTED** [review]
This is an inappropriate addition location ;  it is for direct IANA / ICANN delegation and you cannot add the entry here 

Please place this lower in the file, in the #private section, alphabetically sorted by org name.

**Comment by OsSav** [comment]
> #private

I apologize for the mistake. I missed this detail. I fixed it.
Alphabetical addition was made under Private.

**Comment by dnsguru** [comment]
This is in the queue for the project volunteers to review in a first-come,
first-served manner as they have time available to donate.

**Comment by OsSav** [comment]
@dnsguru Will this process take too long? According to other requests, there are many more users using our service than other requests. As of now, there are over 2k users. For this reason, it is a bit urgent. Thank you for your understanding.

**Review by yoncu — COMMENTED** [review]
Pull request seems to be appropriate and needs to be prioritized.

File: public_suffix_list.dat [hunk 1]
```diff
@@ -11127,6 +11127,11 @@ radio.fm
 // c.la : http://www.c.la/
 c.la
 
+// can.re : https://nic.can.re/
+// Whois Server : whois.can.re
+// Submitted by OsSav Technology Ltd. : support@ossav.com
```
Comment (simon-friedberger) [inline public_suffix_list.dat:11132]: Sorting is by organization name

**Review by simon-friedberger — CHANGES_REQUESTED** [review]
* [x] Expiration
  * can.re expires 2031-10-27
* [x] DNS _psl entries
* [x] Tests pass
* [ ] Sorting
* [x] Reasoning/Organization description

**Review by simon-friedberger — APPROVED** [review]

**Pull request merged**